sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -341,8 +343,7 @@ export class SessionReceiver {
   }
 
   /**
-   * Renews the lock on the session.
-   * Check the `sessionLockedUntilUtc` property on the reciever for the time when the lock expires.
+   * Renews the lock on the session to be set to 30 seconds from the moment of invocation.
```

Comment:
Is this 30 seconds configurable just like for message lock duration for non sessions? Or is it set in the sdk?

Look at the documentation we have for `renewMessageLock`. We talk about the lock duration set during Queue creation.

#2059 was about if the same applies for session locks. If yes, then the docs for `renewSessionLock` needs to be updated just like `renewMessageLock`